data-import/desktop-csv-import/desktop-csv-import.adoc

```diff
@@ -0,0 +1,420 @@
+You can also run a validation query to return a sample of nodes and review that the properties look accurate.
+
+[source, cypher]
+----
+//validate products loaded correctly
+MATCH (p:Product)
+RETURN p LIMIT 20
+----
+
+image::{img}desktop_import_verify_products.jpg[link="{img}desktop_import_verify_products.jpg",role="popup-link"]
+
+.Orders
+Next, we can load the orders.
+Again, since we feel confident our constraint will catch violations and our CSV file data has no duplicates, we can use the `CREATE` statement to insert the data more efficiently.
+Just as with products, we start with the `LOAD CSV` command, then add Cypher statements and include our data conversions.
+
+[source, cypher]
+----
+LOAD CSV WITH HEADERS FROM 'file:///desktop-csv-import/orders.csv' AS row
+CREATE (o:Order {orderId: toInteger(row.orderID)})
+  SET o.orderDateTime = datetime(replace(row.orderDate,' ','T')), o.shipCountry = row.shipCountry
+RETURN count(o)
+----
+
+We can also run a validation query, as before, to verify the graph data looks correct.
+
+[source, cypher]
+----
+//validate orders loaded correctly
+MATCH (o:Order)
+RETURN o LIMIT 20
+----
+
+image::{img}desktop_import_verify_orders.jpg[link="{img}desktop_import_verify_orders.jpg",role="popup-link"]
+
+.Order-details
+Last, but not least, we create the relationship between the products and the orders.
+Since we expect all of our products and all of our orders to already exist in the graph (that data should have been loaded with the last two files), then we start with `MATCH` to find the existing `Product` and `Order` nodes.
+Then, since we know relationships do not yet exist in the graph, the `CREATE` statement will add the new relationship.
+
+As we found when we ran a count on the order-details file above, there are 2,155 rows in the CSV.
+While this is not a huge number for file imports, we decide to have Cypher periodically commit the data to the database to reduce the memory overhead of the transaction state.
+For this, we can add the `USING PERIODIC COMMIT` clause before the `LOAD CSV` command.
+The default value for periodic commit is 1,000, but for this exercise, we will ask Cypher to commit every *500 rows*.
+You could decrease this number if you have a lot of memory already allocated to other tasks, or if it is limited.
+
+[source, cypher]
+----
+USING PERIODIC COMMIT 500
+LOAD CSV WITH HEADERS FROM 'file:///desktop-csv-import/order-details.csv' AS row
+MATCH (p:Product {productId: toInteger(row.productID)})
+MATCH (o:Order {orderId: toInteger(row.orderID)})
+CREATE (o)-[rel:CONTAINS {quantityOrdered: toInteger(row.quantity)}]->(p)
+RETURN count(rel)
+----
+
+Just as we did above, we can validate the data with the query below.
+
+[source, cypher]
+----
+MATCH (p:Product)-[rel:CONTAINS]-(o:Order)
+RETURN p, rel, o LIMIT 50
+----
+
+image::{img}desktop_import_verify_details.jpg[link="{img}desktop_import_verify_details.jpg",role="popup-link"]
+
+== Wrapping up
```

Comment:
Should we talk about any uses of the data? or perhaps point to the next guide?

Reply:
Pointing to the next guide might be a good idea! Maybe the CSV import guide to learn more details and use cases with LOAD CSV.